scripts/setup.sh

```diff
@@ -1,14 +1,11 @@
 #!/bin/bash
 export MYDIR=`dirname $0`
 
-# Universe top level directory
-export UNIVERSE=$MYDIR/..
+# Playground top level directory
+export UNIVERSE=$MYDIR/../
 
 # CheckerFramework Inference
-export CFI="$JSR308"/checker-framework-inference
-
-# Generic Type Inference Solver
-export SOLVER="$JSR308"/generic-type-inference-solver
+export CFI="$MYDIR"/../../checker-framework-inference
 
 # Dependencies
-export CLASSPATH=$UNIVERSE/bin:$CHECKERFRAMEWORK/dataflow/build:$CHECKERFRAMEWORK/javacutil/build:$CHECKERFRAMEWORK/stubparser/build:$CHECKERFRAMEWORK/framework/build:$CHECKERFRAMEWORK/checker/build:$SOLVER/bin:$CHECKERFRAMEWORK/framework/tests/junit-4.12.jar:$CHECKERFRAMEWORK/framework/tests/hamcrest-core-1.3.jar:$CFI/bin:$CFI/dist/org.ow2.sat4j.core-2.3.4.jar:$JSR308/jsr308-langtools/build/classes:$CLASSPATH
+export CLASSPATH="$UNIVERSE"/bin/src:$CHECKERFRAMEWORK/dataflow/build:$CHECKERFRAMEWORK/javacutil/build:$CHECKERFRAMEWORK/stubparser/build:$CHECKERFRAMEWORK/framework/build:$CHECKERFRAMEWORK/checker/build:$CHECKERFRAMEWORK/framework/tests/junit-4.12.jar:$CHECKERFRAMEWORK/framework/tests/hamcrest-core-1.3.jar:$CFI/bin:$CFI/dist/org.ow2.sat4j.core-2.3.4.jar:$CFI/dist/commons-logging-1.2.jar:$CFI/dist/log4j-1.2.16.jar:./build/libs/universe.jar:$CFI/dist/checker-framework-inference.jar:$CLASSPATH
```

Comment:
I think several of these entries can be removed.
- `"$UNIVERSE"/bin/src` is odd. What is in the `bin/src` directory that should be on the classpath? I think `bin` used to be the output directory of Eclipse. I don't think it needs to be added.
- The `build` directories are probably not correct: the class files are further down in the hierarchies. This needs to either be updated or we remove these and just require the .jar files be built.
- are the junit and hamcrest class files not contained in `checker-framework-inference.jar`? I assume the latter contains everything from `checker.jar`, but maybe not these test files.